src/main/java/ui/BoardAutoCreator.java

```diff
@@ -23,6 +25,27 @@
     private static final String MILESTONES = "Milestones";
     private static final String WORK_ALLOCATION = "Work Allocation";
     private static final int MAX_WORK_ALLOCATION_PANELS = 5;
+    public static final String SAMPLE_BOARD = "Sample Board!";
+    private static final String FIRST_SAMPLE_REPO_NAME = "HubTurbo/SampleRepo";
+    private static final String SECOND_SAMPLE_REPO_NAME = "HubTurbo/SampleRepo2";
+    public static final String SAMPLE_BOARD_DIALOG =
+            String.format("%s has been created and loaded.%n%nIt is saved under the name \"%s\".",
+                    SAMPLE_BOARD, SAMPLE_BOARD);
+
+    public static final List<String> SAMPLE_PANEL_NAMES =
+            Collections.unmodifiableList(Arrays.asList(
+                    "All issues in my two sample repos",
+                    "Latest 3 urgent open issues",
+                    "Open issues assigned to Darius or Manmeet"
+            ));
+
+    public static final List<String> SAMPLE_PANEL_FILTERS =
+            Collections.unmodifiableList(Arrays.asList(
+            String.format("(repo:%s | repo:%s) is:issue sort:updated,comments",
+                    FIRST_SAMPLE_REPO_NAME, SECOND_SAMPLE_REPO_NAME),
+            String.format("repo:%s count:3 is:issue is:open label:\"urgent\"", FIRST_SAMPLE_REPO_NAME),
+            String.format("repo:%s is:issue is:open (as:dariusf || as:codemanmeet)", FIRST_SAMPLE_REPO_NAME)
```

Comment:
I feel like we shouldn't abbreviate `assignee`. Might not be so intuitive to first-time readers.